pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -196,8 +197,11 @@ func IsEnabled(v *bool) bool { return v != nil && *v }
 // IsDisabled will only return true if v is not nil and false
 func IsDisabled(v *bool) bool { return v != nil && !*v }
 
+// IsSet will only return true if v is not nil
+func IsSet(v interface{}) bool { return !reflect.ValueOf(v).IsNil() }
```

Comment:
i'm not sure this function has any benefit, what's wrong with `foo != nil`?

Reply:
It was the function name below that got me thinking about this.  IsSet just reads more naturally to me for cases (further down) where it is:
`IsSet(c.VPC.ClusterEndpoints.PublicAccess) && *c.CPC.ClusterEndpoints.PublicAccess`
vs.
`c.VPC.ClusterEndpoints.PublicAccess != nil && *c.VPC.ClusterEndpoints.PublicAccess)`

I could be convinced though if you feel strongly about it.

Reply:
i do because i think it adds unnecessary overhead (for reading, writing, and running code). I wouldn't know what `IsSet()` does without looking at it.

if the long variable name bothers you you could assign them to temp variables

Reply:
I've got that change teed up for my next batch of updates.

Reply:
Done